.github/PULL_REQUEST_TEMPLATE/default_template.md

```diff
@@ -25,6 +25,9 @@ Fixes #[issue number] by @[issue author]
 - [ ] I added or updated documentation (if applicable).
 - [ ] I tried running the project locally and verified that there are no visible errors.
 - [ ] I ran the DAG documentation generator (if applicable).
```

Comment:
Shouldn't this line be removed?

Reply:
Whoops, thanks! I screwed up some of the git operations I was doing and I accidentally left this behind.
```suggestion
```